kafka-connect-azure/src/test/java/io/confluent/connect/Secrets.java

```diff
@@ -0,0 +1,7 @@
+package io.confluent.connect;
+
+public class Secrets {
```

Comment:
I'm not seeing immediately seeing where this class is used.

Reply:
It was used previously in an integration test but I switched to environment variables so I don't accidentally commit secrets. Will remove this file.